docs/how_to/parameterization.md

```diff
@@ -106,6 +118,26 @@ find_replace:
           - "/Hello World.Notebook/notebook-content.py"
           - "\\Goodbye World.Notebook\\notebook-content.py"
 
+    # With enable_deployment_variables feature
```

Comment:
Change each item_name value "Hello World Subfolder" to "Goodbye World" for consistency